common/src/com/intellij/plugins/haxe/util/HaxeEventLogUtil.java

```diff
@@ -0,0 +1,83 @@
+/*
+ * Copyright 2018 Eric Bishton
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package com.intellij.plugins.haxe.util;
+
+import com.intellij.notification.Notification;
+import com.intellij.notification.NotificationType;
+import com.intellij.notification.Notifications;
+import com.intellij.openapi.application.ApplicationManager;
+
+/**
+ * Sends user notifications to the event log.
+ */
+public class HaxeEventLogUtil {
+
+  private HaxeEventLogUtil() {}
```

Comment:
I figured we would be using this feature more in the future (at least I plan to), so I made a class to make accessing the Event Log pane a bit easier.